### DISCUSSION TITLE
Solltemperatur als Entität?

### DISCUSSION THREAD

**Discussion by Lumbeseckel** [SukramJ/aiohomematic#1319] [Q&A] [answered]
Hi,
erst einmal vielen Dank für diese super tolle Integration! 

Eine Frage hätte ich: Ich bräuchte die eingestellte Solltemperatur eines e-TRVs als eigene Entität. WIe kann ich an diese rankommen? Bietet die Integration das überhaupt?

**Comment by stuckis6er** [comment, marked as answer]
Hallo,

in HA unter Einstellungen/ Geräte&Dienste / Helfer kann man einen Templatesensor erstellen.

![Screenshot_20231209-235952_Home Assistant.jpg](https://github.com/danielperna84/hahomematic/assets/52535269/74c760aa-fc00-453a-a28f-a2f9376d611b)

**Reply by Lumbeseckel** [reply]
Aah, vielen Dank! :)
